datalad/support/gitrepo.py

```diff
@@ -1999,6 +1999,36 @@ def tag(self, tag):
             '', ['git', 'tag', str(tag)]
         )
 
+    def get_tags(self, names_only=False):
```

Comment:
note to ourselves -- we might later want to standardize such parameter.  Eg. for `AnnexRepo.whereis` some time ago I added `output` option (return is a keywork) which could be from ` {'descriptions', 'uuids', 'full'}`...